### PR TITLE
[Satya] updates setup instructions for ci-cd course

### PR DESCRIPTION
**Pull request by satya-prakash-23**
[Satya] updates setup instructions for ci-cd course

### CODE REVIEW

File: ci-cd/Setup.md [hunk 1]
```diff
@@ -7,15 +6,34 @@
...
+
+### OS: Windows 7+
+### JDK Setup
+- [JDK 8 Download](http://www.oracle.com/technetwork/pt/java/javase/downloads/jdk8-downloads-2133151.html?printOnly=1)
+    - 64 bit OS : jdk-8u171-windows-x64.exe
+    - 32 bit OS : jdk-8u171-windows-i586.exe
+
+### Maven Setup
+- [Maven Download](http://apache.mirrors.pair.com/maven/binaries/)
+    - apache-maven-3.2.2-bin.zip
```
Comment (mehta-akshay) [inline ci-cd/Setup.md]: Why specifying an older version. Either don't specify or say x or newer version

Comment (mehta-akshay) [reply ci-cd/Setup.md]: And if you are using for your own why not using latest?

File: ci-cd/Setup.md [hunk 2]
```diff
@@ -7,15 +6,34 @@
...
+### IDE Setup [ IntelliJ Community Edition ]
+- [Community Edition Download](https://www.jetbrains.com/idea/download/#section=windows)
+
+### Git Setup [ Command Line ]
+- [Git for Windows Download](https://git-scm.com/download/win)
+
+### Jenkins Setup [ Continuous Integration Build Tool ]
+- [Jenkins Download & Install Instructions](https://jenkins.io/doc/pipeline/tour/getting-started/)
+
+### TOMCAT Setup
```
Comment (mehta-akshay) [inline ci-cd/Setup.md]: We should be using Embedded Tomcat to avoid this to them

File: react/Setup.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+### Course Structure:
+- REACT
```
Comment (mehta-akshay) [inline react/Setup.md:2]: You mean react.js?

**Review by mehta-akshay — CHANGES_REQUESTED** [review]

File: ci-cd/Setup.md (hunk 1 above)
Comment (satya-prakash-23) [reply ci-cd/Setup.md]: Agree. Changed this to apache-maven-3.5.3-bin.zip

File: ci-cd/Setup.md (hunk 2 above)
Comment (satya-prakash-23) [reply ci-cd/Setup.md]: Agree I changed TOMCAT to SpringBoot 2.0.2 GA Release

File: react/Setup.md (hunk 3 above)
Comment (satya-prakash-23) [reply react/Setup.md:2]: Yes just a placeholder for now.

**Comment by satya-prakash-23** [comment]
Changes made as per PR review comments. Please verify.

**Comment by satya-prakash-23** [comment]
@9970807747 Changes made as per PR review comments. Please verify.

**Review by mehta-akshay — APPROVED** [review]

**Pull request merged**